### ISSUE TITLE
Uptime reporting discrepancy

### ISSUE THREAD

**Issue by kobigurk** [MinaProtocol/mina#8991]
We’ve been battling a low reported uptime on the Mina uptime dashboard for a few weeks for our block producer B62qrHzjcZbYSsrcXVgGko7go1DzSEBfdQGPon5X4LEGExtNJZA4ECj, and have recently upgraded our monitoring infrastructure to mitigate more potential issues.

We now have 3 servers with the same block producer key, as allowed by the delegation policy, each of them an up to date version of the sidecar, with upgraded monitoring since 2 days ago.

We see that there are no missed blocks when combining the reports of the 3 servers, as can be seen in the attached files, that show blocks that the sidecar missed on each server. On the other hand, the uptime dashboard keeps reporting around 40.5% uptime, which rises throughout the day and falls afterwards, which doesn’t coincide with what we see.

Our main problem has been that until a few weeks ago we’ve been running the 1.1.5 sidecar version, which had its problems with getting stuck, and we have yet to recover from that period. Even before the upgraded monitoring infrastructure, our internal grafana dashboard shows >90% uptime (probably much more than that).

We’d appreciate help in debugging what is seen on the foundation’s side, as the results seem to conflict and could result in us losing the foundation delegation while being diligent enough to maintain a block producer with great uptime
[2.txt](https://github.com/MinaProtocol/mina/files/6604972/2.txt)
[1.txt](https://github.com/MinaProtocol/mina/files/6604973/1.txt)
[3.txt](https://github.com/MinaProtocol/mina/files/6604974/3.txt)

.

**Comment by EmrePiconbello** [comment]
To support this I get output from our sidecar. Looks like we were running an old version with one of our nodes but There was a planned swap we had for infra change we had some downtime in our control while there was +12 hour to our next slot outside that we only had one time get close to being 3 node out of sync at once, Here is our output,
[node2.txt](https://github.com/MinaProtocol/mina/files/6612338/node2.txt)
[node3.txt](https://github.com/MinaProtocol/mina/files/6612339/node3.txt)
[node1.txt](https://github.com/MinaProtocol/mina/files/6612340/node1.txt)

**Comment by kobigurk** [comment]
Checked the progress in another day, and still no intersection among the missing blocks in the 3 servers.

**Comment by Daeinar** [comment]
@EmrePiconbello So just to clarify, you see a similar uptime discrepancy for your nodes between the locally observed value and the one reported on the official scoreboard? Can you quantify?

**Comment by EmrePiconbello** [comment]
@Daeinar Yes from our monitoring we had no downtime outside the stated time frame we gave which one of our backup server located on gcp cover for a while then we take it down to observe how much we drop. There was sidecar getting stuck issues as well earlier but all of these solved and we are closely monitoring. There is nearly no time our 2 out of 3 nodes we have out of sync and we still keep dropping points I clean the sidecar to provide better data points for the upcoming few days.

**Comment by EmrePiconbello** [comment]
![image](https://user-images.githubusercontent.com/54602437/121447879-8347ec80-c98e-11eb-8284-2c8d4652537b.png)

Here it reports we lost 2 points and we drop a percentage while this is our sidecar reading showing 0 blocks between 3 nodes.
[N3.txt](https://github.com/MinaProtocol/mina/files/6627699/N3.txt)
[N2.txt](https://github.com/MinaProtocol/mina/files/6627700/N2.txt)
[N1.txt](https://github.com/MinaProtocol/mina/files/6627701/N1.txt)

**Comment by kobigurk** [comment]
Uptime continues to be high, as shown by no missing blocks reported by sidecar across 3 servers and blocks keep being produced, while score drops.
